notebooks/MAST/astroqueryMAST/beginner_search_astroqueryMAST/beginner_search_astroqueryMAST.ipynb

```diff
@@ -0,0 +1,395 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astroquery.mast import Observations\n",
+    "\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import SymLogNorm\n",
+    "%matplotlib inline "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Three Ways to Search for MAST Observations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. By Region\n",
+    "You can use `query_region` to find objects based on coordinates and radius.  \n",
+    "The coordinates can be given as a string or `astropy.coordinates` object, and the radius as a string or float object. If no radius is specified, the default is 0.2 degrees."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This will give a warning that the coordinates are being interpreted as an ICRS coordinate provided in degrees\n",
+    "obsByRegion = Observations.query_region(\"322.49324 12.16683\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# How many results? Let's look at the first ten.\n",
+    "print(\"Number of results:\",len(obsByRegion))\n",
+    "print(obsByRegion[:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. By Object Name\n",
+    "\n",
+    "The `query.object` method allows you to search based on object name and optional radius argument.  \n",
+    "The object name is first resolved to sky coordinates, and then the resolved region is queried as above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsByName = Observations.query_object(\"M51\",radius=\".005 deg\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Number of results:\",len(obsByName))\n",
+    "print(obsByName[:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. By Other Criteria (with or without name/region) <a id=\"crit\"></a>\n",
+    "\n",
+    "To search for observations based on additonal parameters, you can use `query_criteria`. In a sense, this is a more powerful version of the tools above, as you can still search by coordinates and objectname; however, you can inculde additional desired criteria. You may also run the search without specifying a name or coordinates.\n",
+    "\n",
+    "To perform the search, give your critera as keyword arguments. Valid criteria are listed [here](https://mast.stsci.edu/api/v0/_c_a_o_mfields.html). Some examples are: \"coordinates\", \"t_exptime\" (exposure time), and \"s_dec\" (source declination)."
```

Comment:
For the examples, I would highlight: filters, instrument_name, t_exptime, provenance_name (useful when limiting searches to specific High Level Science Produts), sequence_number (defined with values corresponding to Kepler Quarters, K2 Campaigns, and TESS Sectors).  I would not reference coordinate info since already we dicsuss them in the above searches, and often people want to use them for cone searches anyways.